bazel/README.md

```diff
@@ -32,19 +32,14 @@ independently sourced, the following steps should be followed:
 As a developer convenience, a [WORKSPACE](https://github.com/envoyproxy/envoy/blob/master/WORKSPACE) and
 [rules for building a recent
 version](https://github.com/envoyproxy/envoy/blob/master/bazel/repositories.bzl) of the various Envoy
-dependencies are provided. These are provided as is, they are only suitable for development and
-testing purposes. The specific versions of the Envoy dependencies used in this build may not be
-up-to-date with the latest security patches. See
-[this doc](https://github.com/envoyproxy/envoy/blob/master/bazel/EXTERNAL_DEPS.md#updating-an-external-dependency-version)
-for how to update or override dependencies.
```

Comment:
Why is this removed? It might be debatable whether this should be there or not, but I think that is a quite distinct PR.

Reply:
The statements are no longer true, we don't have ci/WORKSPACE and prebuilt dependencies anymore.

Reply:
The broader point that we don't necessarily vouch for the security patch status of external dependencies in OSS is true though. We will do best effort, but we are not a Linux distribution. I would like to retain some caveat or warning here.

Reply:
I don't feel this is right place and we actually use those deps for official release so just delete, but that's a separate issue from here.